src/components/PriorAuth/config.json

```diff
@@ -6,7 +6,7 @@
     },
     {
       "name": "Localhost",
-      "url": "http://localhost:9000/fhir"
+      "url": "https://localhost:9000/fhir"
```

Comment:
Maybe add a note that http doesn't work, only https.

Reply:
There's a note on the PAS Readme about this: https://github.com/HL7-DaVinci/prior-auth#ssl-certificates